tests/integration_test.rs

```diff
@@ -32,3 +32,54 @@ fn test_eval() {
     assert_eq!(ui.eval("throw 'ERROR'").unwrap_err(), "ERROR");
     assert!(ui.eval("dtyfhgxnt*").is_err());
 }
+
+#[tokio::test]
```

Comment:
```suggestion
#[test]
```
No need of running this through tokio